flight/Modules/UAVOMavlinkBridge/UAVOMavlinkBridge.c

```diff
@@ -125,7 +123,8 @@ static int32_t uavoMavlinkBridgeInitialize(void) {
 		module_enabled = true;
 		updateSettings();
 
-		serial_buf = PIOS_malloc(MAVLINK_MAX_PACKET_LEN);
+		mav_msg = PIOS_malloc(sizeof(*mav_msg));
```

Comment:
I see. On the one hand, since allocating memory is fundamental to success, and there's no guarantee that we always check in core modules, being conservative and always locking up when a malloc fails on boot seems a good strategy. On the other hand, I agree that we shouldn't force the user to lose all settings just because of a malloc failure.